src/components/table/OcTableFiles.vue

```diff
@@ -316,6 +336,14 @@ export default {
        */
       this.$emit("fileClick", resource)
     },
+
+    $_isResourceClickable(resourceId) {
```

Comment:
Hm... didn't we say we want to skip the `$_` and namespacing as long as we're not in a mixin? 🤔  I get that you needed to have a different name than `isResourceClickable`, because that is the prop name already. Can we rename the prop to `areResourcesClickable`, since it is deciding it for all resources?